examples/platform/efr32/init_otSystem.c

```diff
@@ -65,26 +65,36 @@ void halInitChipSpecific(void);
 void initOtSysEFR(void)
 {
     sl_status_t status;
+    uint32_t SubPrio = 0;
+#if defined(EFR32MG21)
+    // FreeRTOS recommends a lower configuration for CortexM3 and asserts
+    // in some places if using default PRIGROUP_POSITION.
+    SubPrio = 1;
+#endif
 
     __disable_irq();
 
 #undef FIXED_EXCEPTION
 #define FIXED_EXCEPTION(vectorNumber, functionName, deviceIrqn, deviceIrqHandler)
 #define EXCEPTION(vectorNumber, functionName, deviceIrqn, deviceIrqHandler, priorityLevel, subpriority)                            \
-    NVIC_SetPriority(deviceIrqn, NVIC_EncodePriority(PRIGROUP_POSITION, priorityLevel, subpriority));
+    NVIC_SetPriority(deviceIrqn, NVIC_EncodePriority(PRIGROUP_POSITION - SubPrio, priorityLevel, subpriority));
```

Comment:
Should we store `PRIGROUP_POSITION - SubPrio` in a variable instead of duplicating it?